dbt/include/duckdb/macros/materializations/incremental.sql

```diff
@@ -41,11 +43,16 @@
   {% else %}
     {% if language == 'python' %}
       {% set build_python = create_table_as(False, temp_relation, compiled_code, language) %}
+      -- drop the temp relation after we insert the incremental data into the target relation
+      {% set need_drop_temp = True %}
```

Comment:
Ah this looks like a bug fix-- do I have that right? If so, thank you!

Reply:
It's not, I introduced `need_drop_temp` here! When using `CREATE TEMPORARY TABLE` the table is dropped automatically when the `duckdb` database is closed so there's no need to drop it in the macro.